packages/cozy-harvest-lib/src/components/AccountSelectBox/AccountSelectorHeader.tsx

```diff
@@ -18,27 +20,40 @@ export const AccountSelectorHeader = ({
   accountsAndTriggers,
   pushHistory,
   replaceHistory
-}: AccountSelectorHeaderProps): JSX.Element => (
-  <>
-    <DialogBackButton
-      onClick={(): void => replaceHistory(`/accounts/${account._id}`)}
-    />
-    <DialogTitle
-      className="dialogTitleWithBack dialogTitleWithClose"
-      disableTypography
-    >
-      <AccountSelectBox
-        loading={!account}
-        selectedAccount={account}
-        accountsAndTriggers={accountsAndTriggers}
-        onChange={(option: { account: { _id: string } }): void => {
-          pushHistory(`/accounts/${option.account._id}`)
-        }}
-        onCreate={(): void => {
-          pushHistory('/new')
-        }}
-        variant="big"
+}: AccountSelectorHeaderProps): JSX.Element => {
+  // @ts-expect-error IDK
```

Comment:
nit: DialogContext should be easily converted to TS, maybe we can try to do it so we don't have to add those ignore comments?

But if we do this we will only be able to partially type the context as it also contain some cozy-ui values that are not typed (useCozyDialog).